Jenkinsfile

```diff
@@ -197,10 +196,7 @@ pipeline {
             when {
                 beforeAgent true
                 // expression { skipTest != true }
-                expression {
-                    sh script: 'git show -s --format=%B | grep "^Skip-build: true"',
-                       returnStatus: true
-                }
+                expression { ! commitPragma(pragma: 'Skip-build').contains('true') }
```

Comment:
Hrm.  Interesting that simpler _equation_ logic cannot be used here.

Reply:
It probably could if we know the exact string that the shell script run by commitPragma returned.

I suspect a trailing '\n' is present on the return value.  I think your idea of modifying commitPragma to be able to return a boolean value is the long term solution.

Reply:
Having a variant that does return boolean is useful, to be sure, but there will be commit pragmas that we will just want to get the value of, such as `Test-tag:` for example.